students/20th/yujinlim/js/login.js

```diff
@@ -0,0 +1,46 @@
+const loginInputs = Array.from(
+  document.getElementsByClassName("login-form__input")
+);
+const submitBtn = document.querySelector(".login-form__button");
+const OPACITY = "login-form__button--opacity";
+let isIdExists;
+let isPwExists;
+
+function activateSubmitBtn(isEveryInputValueExists) {
+  if (isEveryInputValueExists) {
+    submitBtn.classList.remove(OPACITY);
+    submitBtn.disabled = false;
+    return;
+  }
+
+  if (!isEveryInputValueExists) {
+    submitBtn.classList.add(OPACITY);
+    submitBtn.disabled = true;
+    return;
+  }
+}
+
+function checkInputValue(e) {
+  const inputValue = e.target.value;
+  const inputType = e.target.type;
+
+  if (inputType === "email") {
+    inputValue ? (isIdExists = true) : (isIdExists = false);
+  }
+  if (inputType === "password") {
+    inputValue ? (isPwExists = true) : (isPwExists = false);
+  }
+
+  const isEveryInputValueExists = isIdExists && isPwExists ? true : false;
+
+  activateSubmitBtn(isEveryInputValueExists);
+}
```

Comment:
```suggestion
function checkInputValue(e) {
  const { value, type } = e.target;
  
  const isIdExist = value && (type === "email")
  const isPwExist = value && (type === "password")
  const isEveryInputValueExists = isIdExists && isPwExists;

  activateSubmitBtn(isEveryInputValueExists);
}
```
- `구조분해 할당` 문법을 활용하면 e.target 객체 이하의 값들을 바로 변수로 할당해줄 수 있습니다
- 연산 자체로 boolean 값을 갖게 되는 성질을 이용하면 삼항 연산자를 사용하지 않을 수 있습니다! 위 예제 코드 참고해주세요

Reply:
와 새로운거 배워갑니다~! 말씀해주신 두 가지 주말동안 공부해보고 블로그로도 한번 더 정리하겠습니다. 감사합니다 :)